packages/cozy-sharing/src/components/ShareModal.jsx

```diff
@@ -86,7 +86,7 @@ export const ShareModal = ({
                 createContact={createContact}
                 onShare={onShare}
                 needsContactsPermission={needsContactsPermission}
-                sharingType={sharingType}
+                sharing={sharing}
```

Comment:
Does the DumbShareByEmail already can handle sharing ?

Reply:
I think this is solved in the next commit, the next commit could be fixed up.

Reply:
Hard to squash commits together here. I prefer to not spend to much time by splitting & rewriting them. This commit was before we removed the concept of sharingType and brings few things. It reflects the path I followed to resolve the issue

Reply:
> It reflects the path I followed to resolve the issue

Yes I can see that :D 

By doing so it's harder for the reviewer to understand what has been done and the review quality is poorer as a result.